### PR TITLE
ref(feedback): add crash report icon in feedback list

### LINKED ISSUE #59795: [UI Tweak for User Feedback] Add an icon only for error-related feedbacks and remove error level icons from all feedbacks
Today, our Feedback List page looks like this:

The error level (blue icons in the photo) makes it appear as if the individual feedback is unread, but it's just the error level.

It also distracts from our actual "unread" indicator (purple dot). 

As a user, it's difficult for me to immediately recognize _which_  feedbacks are from the crash report modal and which are not on the List page.

Image

## Proposed Solution

- [x] Drop error level icons from all feedbacks
- [x] Use a unique icon for feedbacks from crash report modal that shows up on the feedback list page.
- [x] Incorporate error level icons on crash-report feedbacks in some fashion

### PR DESCRIPTION
**Pull request by michellewzhang**
closes https://github.com/getsentry/sentry/issues/59795
SCR-20231117-irls

### CODE REVIEW

File: static/app/components/feedback/list/feedbackListItem.tsx [hunk 1]
```diff
@@ -78,7 +80,7 @@ const FeedbackListItem = forwardRef<HTMLDivElement, Props>(
               <FeedbackItemUsername feedbackIssue={feedbackItem} detailDisplay={false} />
             </span>
           </TextOverflow>
-          <span style={{gridArea: 'time'}}>
+          <span style={{gridArea: 'time', textAlign: 'right'}}>
```
Comment (billyvg) [inline static/app/components/feedback/list/feedbackListItem.tsx:83]: Let's make a styled component for this and reserve inline styles for grid-area and dynamic styles

File: static/app/components/feedback/list/feedbackListItem.tsx [hunk 2]
```diff
@@ -89,29 +91,52 @@ const FeedbackListItem = forwardRef<HTMLDivElement, Props>(
...
             <TextOverflow>{feedbackItem.metadata.message}</TextOverflow>
           </div>
-          <div style={{gridArea: 'assigned', display: 'flex', justifyContent: 'end'}}>
+          <div
+            style={{
+              gridArea: 'icons',
+              display: 'flex',
+              justifyContent: 'end',
+              gap: `${space(0.5)}`,
+            }}
```
Comment (billyvg) [inline static/app/components/feedback/list/feedbackListItem.tsx:100]: Same here irt inline styles

File: static/app/components/feedback/list/feedbackListItem.tsx [hunk 3]
```diff
@@ -43,6 +44,7 @@ const FeedbackListItem = forwardRef<HTMLDivElement, Props>(
     const organization = useOrganization();
     const isOpen = useIsSelectedFeedback({feedbackItem});
     const hasReplayId = useFeedbackHasReplayId({feedbackId: feedbackItem.id});
+    const crashReportId = feedbackItem.metadata.source === 'crash_report_embed_form';
```
Comment (billyvg) [inline static/app/components/feedback/list/feedbackListItem.tsx:47]: Ah this isn't related to ids at all right?

```suggestion
    const isCrashReport = feedbackItem.metadata.source === 'crash_report_embed_form';
```

**Review by billyvg — APPROVED** [review]

File: static/app/components/feedback/list/feedbackListItem.tsx (hunk 3 above)
Comment (michellewzhang) [reply static/app/components/feedback/list/feedbackListItem.tsx:47]: oh yep forgot about this!

**Pull request merged**